lib/debug/session.rb

```diff
@@ -186,9 +186,10 @@ def wait_command_loop tc
       @tc = nil
     end
 
+    require 'json' if ENV['RUBY_DEBUG_TEST_MODE']
```

Comment:
In general (without any reasons), `require` should be located at the beginning of file and it make easy to know what kind of libraries are uses in this file.

Reply:
Thanks, I put `require` in beginning of file.